Autopilot/PathManager/Inc/pathModeSelector.hpp

```diff
@@ -75,7 +95,7 @@ class PathModeSelector {
          * 
          * @return none
          */
-        void setAltitdeAirspeedInput(AltitudeAirspeedInput_t alt_airspeed_input);
+        void setAltitudeAirspeedInput(AltitudeAirspeedInput_t alt_airspeed_input);
```

Comment:
nice typo catch. 

In the future though, it would be better if you actually made a ticket for us to edit it instead of fixing it in this branch. Reason why is by fixing this typo you are changing files that are outside the scope of your task, resulting in potential merge conflicts with other PRs in the future. It's not an issue now since you are the only one touching path manager, but something to note for the future when you work on things for WARG and coop

Reply:
ok sounds good!